sequence_processing_pipeline/Commands.py

```diff
@@ -130,3 +130,58 @@ def demux(id_map, fp, out_d, task, maxtask):
     for d in openfps.values():
         for f in d.values():
             f.close()
+
+
+def annotate_filtered_fastq(original_path, stripped_path, output_path):
+    """
+    Annotates a FASTQ file w/missing descriptions using original file.
+    :param original_path: A path to the original, unfiltered file.
+    :param stripped_path: A path to the filtered file.
+    :param output_path: A path for the output file.
+    :return: None
+    """
+    mapping = {}
+
+    # Per FASTQ specification: https://en.wikipedia.org/wiki/FASTQ_format
+    # lines beginning with '@' contain the sequence's identifier and an
```

Comment:
Unfortunately, `@` can also be used in PHRED. It is not safe to rely on `@` as a record identifier

Reply:
Based on https://en.wikipedia.org/wiki/Phred_quality_score I don't yet have an example of a FASTQ file annotated w/PHRED scores, but perhaps a regex of r'^[A-Z]{1,2}\w{8}:\d\d:\w{8}\-\d{4}:\d:\d{4}:\d{4}:\d{4}\/\d' or similar would be suitable?

Reply:
I recommend using a standard well tested parser, such as the one in `skbio` or the readfq approach from Hang Li in `mxdx`

Reply:
sounds good! ty